src/seedu/addressbook/commands/Command.java

```diff
@@ -39,10 +40,7 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
     /**
      * Executes the command and returns the result.
      */
-    public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
-    }
-
+    
```

Comment:
If you delete this method and write it somewhere else, remember to delete the header comment as well. But advice is just to put your abstract method here.